test/modules/linear_boltzmann_solvers/transport_steady/simple_scatter_sgt1000_sigs999.9.xs

```diff
@@ -0,0 +1,13 @@
+# Exported cross section from ChiTech
```

Comment:
ChiTech => OpenSn.

It this still in the code somewhere or just produced with and older version of the code?